apis/datadoghq/v2alpha1/datadogagent_default.go

```diff
@@ -35,17 +35,17 @@ const (
 
 	defaultEBPFCheckEnabled bool = false
 
-	defaultAPMEnabled         bool   = true
-	defaultAPMHostPortEnabled bool   = false
-	defaultAPMHostPort        int32  = 8126
-	defaultAPMSocketEnabled   bool   = true
-	defaultAPMSocketHostPath  string = apicommon.DogstatsdAPMSocketHostPath + "/" + apicommon.APMSocketName
-
-	defaultCSPMEnabled                bool = false
-	defaultCWSEnabled                 bool = false
-	defaultCWSSyscallMonitorEnabled   bool = false
-	defaultCWSNetworkEnabled          bool = true
-	defaultCWSSecurityProfilesEnabled bool = true
+	defaultAPMEnabled                       bool   = true
+	defaultAPMHostPortEnabled               bool   = false
+	defaultAPMHostPort                      int32  = 8126
+	defaultAPMSocketEnabled                 bool   = true
+	defaultAPMSocketHostPath                string = apicommon.DogstatsdAPMSocketHostPath + "/" + apicommon.APMSocketName
+	defaultSingleStepInstrumentationEnabled bool   = false
```

Comment:
```suggestion
	defaultAPMSingleStepInstrEnabled bool   = false
```
- add `APM` in the name
- abbreviate a bit